wal/src/main/java/io/confluent/connect/storage/wal/WAL.java

```diff
@@ -34,4 +34,6 @@
   void close() throws ConnectException;
 
   String getLogFile();
+
+  FilePathOffset extractLatestOffsetFromWAL();
```

Comment:
This is already in WAL, maybe just `extractLatestOffset()` ?

Reply:
good point :)